addon/authenticators/jwt.js

```diff
@@ -123,13 +123,7 @@ export default TokenAuthenticator.extend({
         return reject(new Error('invalid expiration'));
       }
       if (expiresAt > now) {
-        var wait = expiresAt - now - (_this.refreshLeeway * 1000);
-        if (wait > 0) {
```

Comment:
Removed this logic because the restore method should always immediately refresh a token and log the user in rather than schedule a token refresh for a later date.

Correct me if wrong on that though...